libraries/azurerm_mysql_server.rb

```diff
@@ -35,6 +35,7 @@ def initialize(resource_group: nil, server_name: nil)
   end
 
   def firewall_rules
+    management.mysql_server_firewall_rules(@resource_group, @server_name)
```

Comment:
This line should be removed. It has been replaced in a recent PR by the line below.

Reply:
fixed